readinglist/resource.py

```diff
@@ -93,6 +95,12 @@ class RessourceSchema(colander.MappingSchema):
     _id = colander.SchemaNode(colander.String(), missing=colander.drop)
     last_modified = TimeStamp()
 
+    __readonly_fields__ = ('_id', 'last_modified')
```

Comment:
Just a naive inspiration to distinguish this from actual schema fields :/

Any alternative idea ?


```python
class Schema(...):
    field1 = ...
    field2 = ...

    class Options:
        readonly_fields = ...
```